pyllamacpp/model.py

```diff
@@ -75,23 +75,41 @@ def __init__(self,
 
         self._n_ctx = pp.llama_n_ctx(self._ctx)
         self._last_n_tokens = [0] * self._n_ctx  # n_ctx elements
-        self._n_past = 0
+        self.n_consumed_tokens = 0
```

Comment:
So here is just a variable name change. 
I kept the same from the `llama.cpp` algorithm so if someone wants to compare will understand ? 
Why you think we need to call it `n_consumed_tokens` ?